build-aux/modules/90-carla.json

```
@@ -0,0 +1,18 @@
+{
+    "name": "carla",
+    "buildsystem": "cmake-ninja",
+    "builddir": true,
+    "subdir": "cmake",
+    "config-opts": [
+        "-DCARLA_SKIP_LIBDIR_SUFFIX:BOOL=ON",
+        "-DCARLA_USE_JACK:BOOL=OFF",
+        "-DCARLA_USE_OSC:BOOL=OFF"
```

Missing `CMAKE_BUILD_TYPE` options and superfluous type enforcement.

```suggestion
    "config-opts": [
        "-DCMAKE_BUILD_TYPE=Release",
        "-DCARLA_SKIP_LIBDIR_SUFFIX=ON",
        "-DCARLA_USE_JACK=OFF",
        "-DCARLA_USE_OSC=OFF"
```

---

The options come as requirements from OBS folks, see https://github.com/obsproject/obs-deps/pull/189#discussion_r1239936153
Please be consistent on the request for these things.

---

This align with other Flatpak modules that don't have type enforced.

---

sure, but it is then not aligned with how carla is packaged on obs-deps.